crates/subspace-service/Cargo.toml

```diff
@@ -93,4 +93,4 @@ pallet-transaction-payment-rpc-runtime-api = { git = "https://github.com/subspac
 runtime-benchmarks = [
     "dep:frame-benchmarking",
     "sc-domains/runtime-benchmarks",
-]
+]
```

Comment:
Please keep new line at the end of the file and configure IDE to insert it automatically